exchanges/persisted-fetch/src/persistedFetchExchange.ts

```diff
@@ -72,6 +77,8 @@ export const persistedFetchExchange: Exchange = ({
               },
             };
 
+            operation.context.preferGetMethod = !!(options as PersistedFetchExchangeOptions)
```

Comment:
This isn't reset later on, so all queries become GET after this if the option is turned on.
Instead we could add another argument to `makePersistedFetchSource` to "force GET" and then pass it only here, https://github.com/FormidableLabs/urql/blob/c98cb111e0e14a88ebde25475f188cb36b4a7e90/exchanges/persisted-fetch/src/persistedFetchExchange.ts#L120-L123